### DISCUSSION TITLE
Installation issues.

### DISCUSSION THREAD

**Discussion by calarndt** [ericaltendorf/plotman#527] [Q&A] [answered]
annoying but not fatal...
plotman assumes the user name (later found in config) to be chia
yet never mentions it anywhere in the website or in the readme or…

Damn near fatal...
Multiple copies of the config file:
chia@xxxxxxxxx:~$ find . -name plotman.yaml
./chia/.local/config/plotman.yaml
./.config/plotman/plotman.yaml
./.local/config/plotman.yaml
./.local/lib/python3.8/site-packages/plotman/resources/plotman.yaml

FATAL:
Undecipherable error:
the following error is undecipherable since the directory shown in the error is a relative path and
as such one has no method to correct… How would one guess the absolute path???
chia@xxxxxxxxxx:~$ plotman plot
…starting plot loop
Traceback (most recent call last):
File “/home/chia/.local/bin/plotman”, line 8, in
sys.exit(main())
File “/home/chia/.local/lib/python3.8/site-packages/plotman/plotman.py”, line 145, in main
wait_reason = manager.maybe_start_new_plot(cfg.directories, cfg.scheduling, cfg.plotting)
File “/home/chia/.local/lib/python3.8/site-packages/plotman/manager.py”, line 163, in maybe_start_new_plot
p = subprocess.Popen(plot_args,
File “/usr/lib/python3.8/subprocess.py”, line 854, in init
self._execute_child(args, executable, preexec_fn, close_fds,
File “/usr/lib/python3.8/subprocess.py”, line 1702, in _execute_child
raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: ‘chia’

**Comment by mruckli** [comment, marked as answer]
Is it safte to assume that you have left this line in the config to be default and hence the chia folder cannot be found? https://github.com/ericaltendorf/plotman/blob/1876d4cbda73fb38584f95113a00f01af1e21704/src/plotman/resources/plotman.yaml#L18

**Reply by calarndt** [reply]
I don't know but I'm going back over there now and I will check.

**Reply by calarndt** [reply]
mruckli
You nailed it. I also learned from the https://pypi.org/project/plotman/ site that Plotman assumes that a functioning Chia installation! IE it must be activated (. ./actvate) which is also unclear from this github site. my biggest issues were that I didn't even know of this discussion page until long after I stumbled through the install. I know have 10 fully functional installs and am making twice as many plots as before plotman install. There definitely needs to be a clear and complete INSTALL file in this package.

**Reply by calarndt** [reply]
Thanks for the help and insight!
